src/plot_api/plot.js

```diff
@@ -0,0 +1,1483 @@
+/**
+* Copyright 2012-2015, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+// Vendor
+var d3 = require('d3');
+
+// Plotly Components
+var Annotations = require('../components/annotations');
+var Color = require('../components/color');
+var Drawing = require('../components/drawing');
+var ErrorBars = require('../components/errorbars');
+var Legend = require('../components/legend');
+var manageModeBar = require('../components/modebar/manage');
+var Shapes = require('../components/shapes');
+
+// Plotly Plots
+var Plots = require('../plots/plots');
+var Axes = require('../plots/cartesian/axes');
+var Fx = require('../plots/cartesian/graph_interact');
+var Geo = require('../plots/geo/geo');
+var Gl3dLayout = require('../plots/gl3d/layout');
+var Micropolar = require('../plots/polar/micropolar');
+var Scene = require('../plots/gl3d/scene');
+var Scene2D = require('../plots/gl2d/scene2d');
```

Comment:
Down the road, the plot and component modules will be part of a *registry* similar to the current trace modules so that `plot.js` won't have to require them.